packages/components/src/Checkbox/Checkbox.rebuilt.tsx

```diff
@@ -23,6 +23,7 @@ export const CheckboxRebuilt = forwardRef(function CheckboxRebuiltInternal(
   }: CheckboxRebuiltProps,
   ref: React.Ref<HTMLInputElement>,
 ) {
+  const descriptionIdentifier = useId();
```

Comment:
using the same prop name as the existing `descriptionIdentifier` in the FormField component

Reply:
Note that Radio uses a _slightly_ different approach, happy to lean one way or the other: reviewer's choice!

![image](https://github.com/user-attachments/assets/e8c0e26f-e253-401e-9d30-e2b3388a09f9)